src/game-cards/index.ts

```diff
@@ -0,0 +1,99 @@
+import IGameCard from './game-card.interface';
+import { showCart, addToCart, checkCart } from '../utilites/cart';
+function gameCard({num, cover, name, publisher, price, year, platform, internet, genre, popular}: IGameCard) {
+    const dataArr = [
```

Comment:
you can create a separate file for variables which you need. Keep function simple and readable